developer/src/kmc-ldml/src/compiler/messages.ts

```diff
@@ -26,19 +26,19 @@ export class CompilerMessages {
 
   static ERROR_KeyNotFoundInKeyBag = SevError | 0x0005;
   static Error_KeyNotFoundInKeyBag = (o:{keyId: string, col: number, row: number, layer: string, form: string}) =>
-     m(this.ERROR_KeyNotFoundInKeyBag, `Key '${def(o.keyId)}' in position #${def(o.col)} on row #${def(o.row)} of layer ${def(o.layer)}, form '${def(o.form)}' not found in key bag`);
+    m(this.ERROR_KeyNotFoundInKeyBag, `Key '${def(o.keyId)}' in position #${def(o.col)} on row #${def(o.row)} of layer ${def(o.layer)}, form '${def(o.form)}' not found in key bag`);
 
   static HINT_OneOrMoreRepeatedLocales = SevHint | 0x0006;
   static Hint_OneOrMoreRepeatedLocales = () =>
-    m(this.HINT_OneOrMoreRepeatedLocales, `After minimization, one or more locales is repeated and has been removed`);
+  m(this.HINT_OneOrMoreRepeatedLocales, `After minimization, one or more locales is repeated and has been removed`);
 
   static ERROR_InvalidFile = SevError | 0x0007;
   static Error_InvalidFile = (o:{errorText: string}) =>
-    m(this.ERROR_InvalidFile, `The source file has an invalid structure: ${def(o.errorText)}`);
+  m(this.ERROR_InvalidFile, `The source file has an invalid structure: ${def(o.errorText)}`);
```

Comment:
Is this a newer linting thing?  I personally prefer to indent here, since it's a continuation of the following line, but maybe there's been something established recently that I should be made aware of.

If we used brackets in the definition, it would certainly be indented:

```typescript
static Error_InvalidFile = (o:{errorText: string}) => {
  m(this.ERROR_InvalidFile, `The source file has an invalid structure: ${def(o.errorText)}`);
}
```